internal/component/prometheus/exporter/cloudwatch/config.go

```diff
@@ -265,10 +287,11 @@ func toYACEMetrics(ms []Metric, jobNilToZero *bool) []*yaceConf.Metric {
 			// by Alloy every so often, dictated by the scrapedInterval, CloudWatch should return a single datapoint
 			// for each requested metric. That is if Period >= Length, but is Period > Length, we will be getting not enough
 			// data to fill the whole aggregation bucket. Therefore, Period == Length.
+			// NOTE: Period > Length is not a valid configuration and will cause an error. See https://github.com/prometheus-community/yet-another-cloudwatch-exporter/blob/292db29c1537af84a5e831b007bc9ff501708eaa/pkg/config/config.go#L390
```

Comment:
I think we can drop this comment since it's covered in the nicely detailed configuration breakdown